apps/web/src/components/auth/LoginForm.tsx

```diff
@@ -0,0 +1,119 @@
+import { useContext } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { useMutation } from 'react-query';
+import styled from '@emotion/styled';
+import { useForm } from 'react-hook-form';
+import * as Sentry from '@sentry/react';
+import { Divider, Button as MantineButton, Center } from '@mantine/core';
+import { AuthContext } from '../../store/authContext';
+import { api } from '../../api/api.client';
+import { PasswordInput, Button, colors, Input, Text } from '../../design-system';
+import { Github } from '../../design-system/icons';
+
+type Props = {};
+
+export function LoginForm({}: Props) {
+  const navigate = useNavigate();
+  const { setToken } = useContext(AuthContext);
+  const { isLoading, mutateAsync, isError, error } = useMutation<
+    { token: string },
+    { error: string; message: string; statusCode: number },
+    {
+      email: string;
+      password: string;
+    }
+  >((data) => api.post(`/v1/auth/login`, data));
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({});
+
+  const onLogin = async (data) => {
+    const itemData = {
+      email: data.email,
+      password: data.password,
+    };
+
+    try {
+      const response = await mutateAsync(itemData);
+
+      setToken((response as any).token);
+      navigate('/templates');
+    } catch (e: any) {
+      if (e.statusCode !== 400) {
+        Sentry.captureException(e);
+      }
+    }
+  };
+
+  return (
+    <>
+      <GithubButton
+        my={30}
+        variant="white"
+        fullWidth
+        radius="md"
+        leftIcon={<Github />}
+        sx={{ color: colors.B40, fontSize: '16px', fontWeight: '700', height: '50px' }}>
+        Sign In with Github
+      </GithubButton>
```

Comment:
Is this possible to wrap it, or make it as "a" take and do a link to "API_URL/v1/auth/github"?

Reply:
yes :)